opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationAddStixCoreObjects.jsx

```diff
@@ -194,52 +173,48 @@ const InvestigationAddStixCoreObjects = (props) => {
         title={t_i18n('Add entities')}
         containerRef={containerRef}
       >
-        <UserContext.Consumer>
-          {({ platformModuleHelpers }) => (
-            <ListLines
-              sortBy={sortBy}
-              orderAsc={orderAsc}
-              dataColumns={buildColumns(platformModuleHelpers)}
-              handleSearch={setSearchTerm}
-              keyword={
-                  mapping && searchTerm.length === 0 ? selectedText : searchTerm
-                }
-              handleSort={handleSort}
-              handleAddFilter={handleAddFilter}
-              handleRemoveFilter={handleRemoveFilter}
-              handleSwitchLocalMode={handleSwitchLocalMode}
-              handleSwitchGlobalMode={handleSwitchGlobalMode}
-              disableCards={true}
-              filters={filters}
-              paginationOptions={searchPaginationOptions}
-              numberOfElements={numberOfElements}
-              iconExtension={true}
-              parametersWithPadding={true}
-              disableExport={true}
-              availableEntityTypes={[resolveAvailableTypes()]}
-            >
-              <QueryRenderer
-                query={investigationAddStixCoreObjectsLinesQuery}
-                variables={{ count: 100, ...searchPaginationOptions }}
-                render={({ props: renderProps }) => (
-                  <InvestigationAddStixCoreObjectsLines
-                    data={renderProps}
-                    workspaceId={workspaceId}
-                    paginationOptions={searchPaginationOptions}
-                    dataColumns={buildColumns(platformModuleHelpers)}
-                    initialLoading={renderProps === null}
-                    workspaceStixCoreObjects={workspaceStixCoreObjects}
-                    onAdd={onAdd}
-                    onDelete={onDelete}
-                    setNumberOfElements={setNumberOfElements}
-                    mapping={mapping}
-                    containerRef={containerRef}
-                  />
-                )}
+        <ListLines
+          helpers={helpers}
+          sortBy={sortBy}
+          orderAsc={orderAsc}
+          dataColumns={buildColumns()}
+          handleSearch={helpers.handleSearch}
+          keyword={keyword}
+          handleSort={helpers.handleSort}
+          handleAddFilter={helpers.handleAddFilter}
+          handleRemoveFilter={helpers.handleRemoveFilter}
+          handleSwitchLocalMode={helpers.handleSwitchLocalMode}
+          handleSwitchGlobalMode={helpers.handleSwitchGlobalMode}
```

Comment:
I am not sure about the impact as we had many children components that passed these props but we should try to remove it later.
With `helpers ` we can use all the functions that is related to this..

Reply:
yes it would be nice to refacto this later

Reply:
(we pass those children + 'helpers' everywhere when listLines is called for the moment)

Reply:
this refacto was not done for the moment because there are lots of files to modify and the release is soon